content/en/serverless/step_functions/_index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Serverless Monitoring for AWS Step Functions
+kind: documentation
+further_reading:
+- link: "/serverless/step_functions/installation"
+  tag: "Documentation"
+  text: "Install Serverless Monitoring for AWS Step Functions"
+- link: "/serverless/step_functions/troubleshooting"
+  tag: "Documentation"
+  text: "Troubleshoot Serverless Monitoring for AWS Step Functions"
+- link: "/integrations/amazon_step_functions/"
+  tag: "Documentation"
+  text: "AWS Step Functions integration"
+- link: "/serverless/aws_lambda/"
+  tag: "Documentation"
+  text: "Serverless Monitoring for AWS Lambda"
+---
+<div class="alert alert-warning">This feature is in public beta. You can provide feedback by emailing <u>serverless-product@datadoghq.com</u> with the subject line <i>[Step Function Beta Feedback]</i>.</div>
+
+{{< img src="serverless/step_functions/overview.png" alt="New Log index" style="width:100%;" >}}
```

Comment:
@sumedham is there a better trace that shows multiple interesting steps?

Reply:
Or maybe one showing an interesting error? aka, something immediately interests the first-time reader? maybe something you would use for the product announcement?

Reply:
Yes good call, trying to find one to replace it with